projects/mysadna/src/app/app.component.ts

```diff
@@ -2,7 +2,7 @@ import { Component } from '@angular/core';
 
 import { EncodingService } from './services/encoding.service';
 import { FirebaseService } from './services/firbase.service';
-import { Data } from './proto';
+import { Data } from '@/proto';
```

Comment:
Looks like your project doesn't have `fs-extra`, which is needed to create proto function.
How to fix it:
```
npm i fs-extra --save-dev
```

Reply:
installed.